latex-gitCheatSheet/FSFW_git-Spickzettel.tex

```diff
@@ -94,162 +97,165 @@
 \begin{document}
 \vspace*{-1cm}
 \hfill\includegraphics[width=0.1\textwidth]{../img-src/fsfw-logo-with-text.pdf}
-\vspace*{-1cm}\begin{center}{\large\bfseries Git Cheat Sheet Volume 1}\end{center}
+\vspace*{-1cm}\begin{center}{\large\bfseries Git-Spickzettel Nr. 1}\end{center}
 \begin{multicols}{2}
 %% configure your settings for git
 \colouredbox{yellow}{
-	\begin{center}\section{Configuring git}\end{center}
+	\begin{center}\section{Git konfigurieren}\end{center}
 	\cmdline{git config [-{}-global] [option]}
-	\annotation{with \texttt{-{}-global} it is stored in \texttt{$\tilde{\ }$/.gitconfig}}
-	\subsection{Information about the user}
+	\annotation{mit \texttt{-{}-global}: wird es gespeichert in \texttt{$\tilde{\ }$/.gitconfig}}
```

Comment:
"mit […] wird es in […] gespeichert" liest sich flüssiger als "mit […] wird es gespeichert in […]".